src/http/response.rs

```diff
@@ -2,8 +2,14 @@ use anyhow::Result;
 use derive_setters::Setters;
 use hyper::body::Bytes;
 use serde::de::DeserializeOwned;
+use tonic::Code;
 
-use crate::grpc::protobuf::ProtobufOperation;
+use crate::grpc::protobuf::{ProtobufMessage, ProtobufOperation};
+use crate::lambda::EvaluationError;
+
+pub(crate) static GRPC_STATUS: &str = "grpc-status";
+pub(crate) static GRPC_MESSAGE: &str = "grpc-message";
+pub(crate) static GRPC_STATUS_DETAILS: &str = "grpc-status-details-bin";
```

Comment:
can we use [tonic::Status](https://docs.rs/tonic/0.11.0/tonic/struct.Status.html) here instead of implementing manually?

I think `Status::from_header_map` could be useful